### PR TITLE
fix: use react-query refetch for refreshing calls

### PR DESCRIPTION
**Pull request by nmalzieu**

### CODE REVIEW

File: packages/core/src/hooks/call.ts [hunk 1]
```diff
@@ -102,8 +102,8 @@ export function useStarknetCall<T extends unknown[]>({
 
   return {
     data,
-    loading: isLoading,
-    refresh: () => undefined,
+    loading: isLoading || isRefetching,
```
Comment (nmalzieu) [inline packages/core/src/hooks/call.ts:105]: Not sure if we want the `loading` value to be true when refreshing
We can also add another "refreshing" boolean

Comment (nmalzieu) [reply packages/core/src/hooks/call.ts:105]: Actually I think that's better to add a refreshing boolean - sometimes we don't want to display data until the first load is done but we want to keep displaying it while we refresh. 
What do you think?

Comment (fracek) [reply packages/core/src/hooks/call.ts:105]: Yeah I agree with the second option. That can be done in a new refactor where we change the API to be more like wagmi.

**Comment by fracek** [comment]
Merged in 60aeefb125e68fabeb196026b5e5aeaac11fdb5b, GH fails at detecting merges again :/

File: packages/core/src/hooks/call.ts (hunk 1 above)
Comment (coburn24) [reply packages/core/src/hooks/call.ts]: should 'isrefetching' be 'isrefreshing' and also 'refetch' be 'refresh' or was this done on purpose?

Comment (fracek) [reply packages/core/src/hooks/call.ts]: the old api used `refresh` while react-query calls it `refetch`.

**Pull request closed without merging**